README.md

```diff
@@ -17,16 +17,31 @@ originally created by Brandon Liu for Protomaps.
   - Async `mmap` (Tokio) for local files
   - Async `http` and `https` (Reqwuest + Tokio) for URLs
   - Async `s3` (Rust-S3 + Tokio) for S3-compatible buckets
+- Creating PMTile archives
 
 ## Plans & TODOs
 
 - [ ] Documentation and example code
-- [ ] Support writing and conversion to and from MBTiles + `x/y/z`
+- [ ] Support conversion to and from MBTiles + `x/y/z`
 - [ ] Support additional backends (sync `mmap` and `http` at least)
 - [ ] Support additional async styles (e.g., `async-std`)
 
 PRs welcome!
 
+## Usage examples
+
+### Writing a PMTiles file
+
+```rust
+use pmtiles::{tile_id, PmTilesWriter, TileType};
+use std::fs::File;
+
+let file = File::create("tiles.pmtiles").unwrap();
+let mut writer = PmTilesWriter::new(TileType::Mvt).create(file).unwrap();
+writer.add_tile(tile_id(0, 0, 0), &[/*...*/]).unwrap();
```

Comment:
do we allow zero-length tiles?

Reply:
Zero-length tiles added with `add_tile` are ignored. The spec doesn't allow storing zero-length tiles and I think throwing an error is not absolutely necessary.

Reply:
I agree.